libmscore/hairpin.cpp

```diff
@@ -89,12 +89,12 @@ Element* HairpinSegment::drop(EditData& data)
 void HairpinSegment::layout()
       {
       const qreal _spatium = spatium();
-      const int _track = track();
+      const int _trck = track();
       if (autoplace() && !score()->isPalette()) {
             // Try to fit between adjacent dynamics
             if (isSingleType() || isBeginType()) {
                   Segment* start = hairpin()->startSegment();
-                  Dynamic* sd = start ? toDynamic(start->findAnnotation(ElementType::DYNAMIC, _track, _track)) : nullptr;
+                  Dynamic* sd = start ? toDynamic(start->findAnnotation(ElementType::DYNAMIC, _trck, _trck)) : nullptr;
```

Comment:
or `_track` and `_trck`?

Reply:
the local `_trck' rather than the global `_track`, at least that is what gets taken withoug that change

Reply:
Yes, `_trck` is good, here should be the value returned by `track()` call. Not sure which is the other "global" `_track` variable though.